artifacts/mysql-statefulset.yaml

```diff
@@ -87,19 +90,25 @@ spec:
             memory: 1Gi
         livenessProbe:
           exec:
-            command: ["mysqladmin", "ping"]
+            command: ["bash", "-c", "mysqladmin ping -p$MYSQL_ROOT_PASSWORD"]
```

Comment:
Note that we might need to surround the variable with apostrophes `'${MYSQL_ROOT_PASSWORD}'` or `"${MYSQL_ROOT_PASSWORD}"` in order for it to work when password has whitespace characters.

Reply:
Done. FYI: ```"${MYSQL_ROOT_PASSWORD}"``` version doesn't work.